yt_dlp/extractor/youtube.py

```diff
@@ -3837,53 +3845,112 @@ def _generate_player_context(cls, sts=None):
             **cls._get_checkok_params(),
         }
 
-    def _get_config_po_token(self, client):
+    def _get_config_po_token(self, client: str, context: _PoTokenContext):
         po_token_strs = self._configuration_arg('po_token', [], ie_key=YoutubeIE, casesense=True)
         for token_str in po_token_strs:
-            po_token_client, sep, po_token = token_str.partition('+')
+            po_token_meta, sep, po_token = token_str.partition('+')
+            if not sep:
+                self.report_warning(f'Invalid po_token configuration format. Expected "CLIENT.CONTEXT+PO_TOKEN", got "{token_str}"', only_once=True)
```

Comment:
the long line hurts me
```suggestion
                self.report_warning(
                    'Invalid po_token configuration format. Expected '
                    f'"CLIENT.CONTEXT+PO_TOKEN", got "{token_str}"', only_once=True)
```

Reply:
fixed